### ISSUE TITLE
Works fine in 1.18.1 but the color applies only to ribbon.

### ISSUE THREAD

**Issue by spinodal23** [Tfarcenim/Enchanted-Book-Redesign-Fabric#6]
On the CurseForge page, the latest version is 1.17, but this mod also works in 1.18.1 without any crashing. No problem at all.
One thing to say is that this mod add colors only to the ribbons, not to the glint. May be this is because I am running this mod in 1.18.1 not in 1.17.
Below is the screen capture.

![2022-01-08_15 09 05](https://user-images.githubusercontent.com/96723542/148633708-1a9e1494-761c-4451-be27-73fd2cffe1bc.png)

**Comment by Tfarcenim** [comment]
That's because post 1.17 I haven't figured out a way to hack the glint yet as that's changed for the 3rd time.

**Comment by spinodal23** [comment]
Ah okay, thank you. Will there be any update to 1.18.1 or not?

**Comment by Tfarcenim** [comment]
Not until I figure out the glint issue, but that's low priority as it would be a big timesink and I have other mods to get to.
